ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -136,7 +135,11 @@ public void sendRequest(
     requestBuilder.headers(requestHeaders);
 
     if (data == null) {
-      requestBuilder.method(method, null);
+      if (method.equals("POST") || method.equals("PUT") || method.equals("PATCH")){
+        requestBuilder.method(method, RequestBody.create(null, new byte[0]));
```

Comment:
@mkonicek Could you clarify why @lexs or how I could use ByteString.EMPTY here instead ?